### PR TITLE
(blog) Pending blog posts catchup

### PR DESCRIPTION
**Pull request by Disha-maker**

### CODE REVIEW

File: _lifeofinternatpuppet/post_24.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+   - [puppetlabs-wsus_client](https://github.com/puppetlabs/puppetlabs-wsus_client/pull/153)
+   - [puppetlabs-websphere_application_server](https://github.com/puppetlabs/puppetlabs-websphere_application_server/pull/218)
+   - [puppetlabs-tomcat](https://github.com/puppetlabs/puppetlabs-tomcat/pull/433)
+
+- I also spent some time understanding how to do if statements in Ruby, links given below.
+   - [Ruby in Twenty Minutes](https://www.ruby-lang.org/en/documentation/quickstart/)
+   - [Ruby Tutorial For Complete Beginners: Learn Ruby Now!](https://www.rubyguides.com/ruby-tutorial/ruby-if-else/)
+   - [Learn Ruby the Hard Way](https://learnrubythehardway.org/book/)
+
+- I was contacted by the DIO Team regarding this [PR](https://github.com/puppetlabs/puppetlabs-modules/pull/10183) raised before the Christmas break. I paired up with Sheena, the issue was resolved and the PR was merged by the DIO Team. Thanks to Sheena and Jack for a follow up!
```
Comment (pmcmaw) [inline _lifeofinternatpuppet/post_24.md:28]: This is a private repo please remove.

File: _lifeofinternatpuppet/post_27.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+I really enjoyed releasing these modules :)
+
+- I was also told by Paula during the 1:1 session that start putting the content of the Google Doc - 'Accessing IAC Training Materials' on Confluence. I also learnt that all training materials are drafted and reviewed on Confluence. Thanks for pointing out Paula :)
+
+- The first draft of Accessing IAC Training Materials on Confluence was ready by the end of the week!
+
+- I also spent time catching up on my blog posts for the 'Life of an Intern at Puppet' blog page for Weeks 17 and 18. The [week 17 PR](https://github.com/puppetlabs/iac/pull/205) was merged by Paula, thank you!
+
+
```
Comment (pmcmaw) [inline _lifeofinternatpuppet/post_27.md:34]: Large gap here I think that may be an accident.

**Comment by pmcmaw** [comment]
2 minor comments, when addressed I will merge :-) @Disha-maker

File: _lifeofinternatpuppet/post_27.md (hunk 2 above)
Comment (Disha-maker) [reply _lifeofinternatpuppet/post_27.md]: This is also fixed in the commit message below, check the files please.

File: _lifeofinternatpuppet/post_24.md (hunk 1 above)
Comment (Disha-maker) [reply _lifeofinternatpuppet/post_24.md]: The paragraph is removed, check the files of the latest commit message pushed to this pr

**Pull request merged**